Documentation/ApiOverview/Events/Events/Redirects/AfterAutoCreateRedirectHasBeenPersistedEvent.rst

```diff
@@ -0,0 +1,41 @@
+..  include:: /Includes.rst.txt
+..  index:: Events; AfterAutoCreateRedirectHasBeenPersistedEvent
+..  _AfterAutoCreateRedirectHasBeenPersistedEvent:
+
+
+============================================
+AfterAutoCreateRedirectHasBeenPersistedEvent
+============================================
+
+..  versionadded:: 12.3
+
+The PSR-14 event
+:php:`\TYPO3\CMS\Redirects\Event\AfterAutoCreateRedirectHasBeenPersistedEvent`
+allows extension authors to react on persisted auto-created redirects. This
```

Comment:
```suggestion
allows extensions to react on persisted auto-created redirects. This
```

Reply:
How can a thing react?

Reply:
How can the extension author react when he is not there?

Reply:
The developer reacts on an event programmatically...

Reply:
I would have said, the developer instructs the extension to react. But whatever. Keep it like this if you want